src/styles/main.scss

```diff
@@ -0,0 +1,61 @@
+@import './utils/variables';
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  margin: 0;
+}
+
+.calendar {
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: $width-calendar;
+  margin: auto;
+
+  @each $name, $number in $week-day {
```

Comment:
The loop uses `$week-day`. Ensure that `$week-day` is a map or list defined in the `variables` file, as it is used to dynamically generate styles.